versioned_docs/version-0.11/tools/Ide/helix.md

```diff
@@ -0,0 +1,72 @@
+---
```

Comment:
you should put the doc into docs/tools/Ide/  and update `index.md`, not in versioned_docs

Reply:
maybe change to https://github.com/helix-editor/helix ?

Reply:
Done!